api/src/main/scala/hmda/api/model/EditResults.scala

```diff
@@ -0,0 +1,11 @@
+package hmda.api.model
+
+case class LarId(loanId: String)
+case class LarEditResult(lar: LarId)
+case class EditResult(edit: String, lars: Seq[LarEditResult])
+case class EditResults(edits: Seq[EditResult])
+case object EditResults {
+  def empty: EditResults = EditResults(Nil)
```

Comment:
This line also isn't used by the tests. So, same question:  Is it an artifact of a previous strategy (and should be removed)? or does it have a use case that isn't covered by tests?